pages/examples/seed-projects.md

```diff
@@ -92,6 +92,11 @@ in the `Dockerfile`.
 __Note:__ This project will not build on any of the ARM architecture board. Currently,
 this project will only work on the Intel Edison.
 
+[Resin Golang Image Example][golang-image-link]
+
+This project contains examples of how to use Resin Golang images on devices supported by Resin.io.
+This project will work on any devices, notice that it requires native ARM builders when building for ARM architecture devices.
+
```

Comment:
@shaunmulligan Updated! I'm happy to bring it here, if it can help our users  :)